### ISSUE TITLE
Is it possible to export the scrobbled data form Maloja?

### ISSUE THREAD

**Issue by Haze-sh** [krateng/maloja#223]
I want to try this service but I am wondering if I can export its data, preferably to JSON format

**Comment by CodeWithMa** [comment]
Yes this is possible. If you log into the admin view on your maloja: https://maloja.example.com/admin_overview you can export your data in JSON format.

**Comment by Haze-sh** [comment]
This makes the project perfect for me! Thank your help.
